packages/common/src/hooks/useSortBy/useSortBy.ts

```diff
@@ -8,6 +8,7 @@ export interface SortRule<T> {
 
 export interface SortBy<T> extends SortRule<T> {
   direction: 'asc' | 'desc';
+  getSortValue?: (row: T) => string | number;
```

Comment:
it's not ideal.. and is here so that I can easily re-sort the data when flipping between grouped/ungrouped. I suppose that the grouping shouldn't be modifying the order of the items and there shouldn't be a need to re-sort.. but the grouping was using the original un-sorted dataset. likewise.. didn't see any good alternatives 😬